tests/CMakeLists.txt

```diff
@@ -11,11 +11,13 @@ file(GLOB sources_utils utilities/*.cc)
 file(GLOB headers_utils utilities/*.h)
 
 add_library(TestDataModel SHARED ${sources} ${sources_utils} ${headers} ${headers_utils})
-target_link_libraries(TestDataModel podio::podio)
+target_link_libraries(TestDataModel podio::podio ROOT::Core)
 target_include_directories(TestDataModel PUBLIC
   ${CMAKE_SOURCE_DIR}/include
-  ${CMAKE_CURRENT_SOURCE_DIR}/
+  ${CMAKE_CURRENT_SOURCE_DIR}
+  ${ROOT_INCLUDE_DIRS}
```

Comment:
Was this really necessary? ROOT::Core should include this already even on old root versions given this in podio/CMakeLists.txt

https://github.com/AIDASoft/podio/blob/6019f0f3eb9c4924f8fd21d48cbbd30aab8784cc/CMakeLists.txt#L63-L65

Reply:
Yes. Without this fix I got:
```
[ 15%] Building CXX object tests/CMakeFiles/TestDataModel.dir/TestDataModel.cxx.o
cd /Users/gaede/podio/podio/build/tests && /Library/Developer/CommandLineTools/usr/bin/c++  -DTestDataModel_EXPORTS -I/Users/gaede/podio/podio/include -I/Users/gaede/podio/podio/tests  -std=c++14 -stdlib=libc++ -O2 -g -DNDEBUG -fPIC   -std=c++14 -o CMakeFiles/TestDataModel.dir/TestDataModel.cxx.o -c /Users/gaede/podio/podio/build/tests/TestDataModel.cxx
/Users/gaede/podio/podio/build/tests/TestDataModel.cxx:13:10: fatal error: 'RConfig.h' file not found
#include "RConfig.h"
         ^~~~~~~~~~~
1 error generated.
```

Reply:
... maybe, because I fixed this first and only then added ROOT:Core ?

Reply:
Probably

Reply:
Indeed - see my last commit.